### PR TITLE
uploading assignments

### PR DESCRIPTION
**Pull request by mubeen2710**
all assignments of mubeen till the date

### CODE REVIEW

**Review by Richa199 — APPROVED** [review]

**Review by Jestinxavier — APPROVED** [review]

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
need to add html tag in every files 
indentation need to be done

File: mubeen/html/assign3/mubeenA3_1.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                <li>Triveni</li>
+                <li>Kalyani</li>
+            </ul>
+            </li>
+        </ol>
+        </li>
+    </ol>
+
+</body>
+</html>
```
Comment (Pawan1005) [inline mubeen/html/assign3/mubeenA3_1.html]: need indentation
add new line at the end of file

File: mubeen/html/assign3/mubeenA3_2home.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+<p> We are living in a digital revolution.  As the technology landscape changes rapidly, more businesses, tools, and technologies are now
+    moving online. As a developer that builds web applications and technologies, you are needed now more than ever. Our full-stack web 
+    development stream helps you master all the required skills and gain valuable experience so that you can launch your career as a web developer.
+   ​Course Flow:
+   ​You will work your way through the foundations i.e. HTML, CSS, and JavaScript.
+   ​Once you've got a grip on the fundamentals, you will be trained to work on web technologies: Angular/Node.js, React, and Mongo. ​Realizing that it's 
+   important to be up to speed with current tech trends, the last unit of your training period will be an intermediate-level course on Kubernetes.</p>
+</div></p>
+</body>
+</html>
```
Comment (Pawan1005) [inline mubeen/html/assign3/mubeenA3_2home.html]: need indentation
add new line at the end of file

File: mubeen/html/assign4/q1.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                </tr>
+                <tr>
+                    <th>Female</th>
+                    <td>1.9</td>
+                    <td>0.003</td>
+                    <td>40</td>
+                </tr>
+            </table>
+</body>
+</html>
```
Comment (Pawan1005) [inline mubeen/html/assign4/q1.html]: need indentation
add new line at the end of file

File: mubeen/html/assign4/q2.html [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        <tr class="a2">
+          <th>4:30</th>
+          <td>&nbsp;</td>
+          <td>&nbsp;</td>
+          <td>&nbsp;</td>
+        </tr>
+      </body>
+    </table>
+  </body>
+</html>
```
Comment (Pawan1005) [inline mubeen/html/assign4/q2.html:127]: need indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**